src/HW/client/src/components/AppContainer/AppContainer.jsx

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState } from "react";
+import "./AppContainer.css";
+import List from "../List/List";
+import ListControls from "../ListControls/ListControls";
+import ListFooter from "../ListFooter/ListFooter";
+import {
+  fetchItems,
+  createItem,
+  deleteItem,
+  updateStatus,
+  editTaskName,
+  deleteAllItems,
+} from "../../Services/ItemClient";
+
+function AppContainer() {
+  const [items, setItems] = useState([]);
+  const [numOfTasks, setNumOfTasks] = useState(0);
+
+  const renderNewItems = async (item) => {
+    try {
+      const newItems = await createItem(item);
+      newItems.forEach((item) => {
+        items.push(item);
+      });
+      setItems([...items]);
+      setNumOfTasks(items.length);
+    } catch (err) {
+      throw new Error(err);
+    }
```

Comment:
redundant, if you only throw the error there is no point in catching the exception